.github/workflows/ci.yml

```
@@ -54,12 +42,63 @@ jobs:
       - name: Run integration tests
         if: matrix.task == 'test:integration'
         run: bun run test:integration
+
+  ci-e2e:
+    runs-on: ubuntu-24.04
+    services:
+      postgres:
+        image: postgres:17
```

nits: 本番サーバではPostgreSQL 18を使ってます
```suggestion
        image: postgres:18
```